node/overseer/src/lib.rs

```diff
@@ -1404,67 +1719,75 @@ impl metrics::Metrics for Metrics {
 				)?,
 				registry,
 			)?,
-			message_relay_timings: prometheus::register(
-				prometheus::Histogram::with_opts(
-					prometheus::HistogramOpts {
-						common_opts: prometheus::Opts::new(
-							"parachain_overseer_messages_relay_timings",
-							"Time spent holding a message in the overseer before passing it to `route_message`",
-						),
-						// guessing at the desired resolution, but we know that messages will time
-						// out after 0.5 seconds, so the bucket set below seems plausible:
-						// `0.0001 * (1.6 ^ 18) ~= 0.472`. Prometheus auto-generates a final bucket
-						// for all values between the final value and `+Inf`, so this should work.
-						//
-						// The documented legal range for the inputs are:
-						//
-						// - `> 0.0`
-						// - `> 1.0`
-						// - `! 0`
-						buckets: prometheus::exponential_buckets(0.0001, 1.6, 18).expect("inputs are within documented range; qed"),
-					}
+			to_subsystem_bounded_sent: prometheus::register(
+				prometheus::GaugeVec::<prometheus::U64>::new(
+					prometheus::Opts::new(
+						"parachain_subsystem_bounded_sent",
+						"Number of elements sent to subsystems' bounded queues",
+					),
+					&[
+						"subsystem_name",
+					],
+				)?,
+				registry,
+			)?,
+			to_subsystem_bounded_received: prometheus::register(
+				prometheus::GaugeVec::<prometheus::U64>::new(
+					prometheus::Opts::new(
+						"parachain_subsystem_bounded_received",
+						"Number of elements received by subsystems' bounded queues",
+					),
+					&[
+						"subsystem_name",
+					],
 				)?,
 				registry,
 			)?,
-			from_overseer_sent: prometheus::register(
+			to_subsystem_unbounded_sent: prometheus::register(
 				prometheus::GaugeVec::<prometheus::U64>::new(
 					prometheus::Opts::new(
-						"parachain_from_overseer_sent",
-						"Number of elements sent by the overseer to subsystems",
+						"parachain_subsystem_unbounded_sent",
+						"Number of elements sent to subsystems' unbounded queues",
 					),
 					&[
 						"subsystem_name",
 					],
 				)?,
 				registry,
 			)?,
-			from_overseer_received: prometheus::register(
+			to_subsystem_unbounded_received: prometheus::register(
 				prometheus::GaugeVec::<prometheus::U64>::new(
 					prometheus::Opts::new(
-						"parachain_from_overseer_received",
-						"Number of elements received by subsystems from overseer",
+						"parachain_subsystem_unbounded_received",
+						"Number of elements received by subsystems' unbounded queues",
 					),
 					&[
 						"subsystem_name",
 					],
 				)?,
 				registry,
 			)?,
-			to_overseer_sent: prometheus::register(
-				prometheus::Gauge::<prometheus::U64>::with_opts(
+			signals_sent: prometheus::register(
+				prometheus::GaugeVec::<prometheus::U64>::new(
 					prometheus::Opts::new(
-						"parachain_to_overseer_sent",
-						"Number of elements sent by subsystems to overseer",
+						"parachain_overseer_signals_sent",
```

Comment:
naming nit: what's the difference between `parachain_subsystem` and `parachain_overseer` prefix? They both are parameterized by `subsystem_name`.

Reply:
`subsystem_signals_sent` sounds like the subsystems are sending the signals but they are actually receiving them. `overseer_unbounded_sent` has the same issue in the opposite direction.

Reply:
These are `OverseerSignals` so `overseer_signals_sent` makes sense I think. But I could rename `subsystem_bounded_sent` etc.. not sure to what